packages/components/src/FormField/FormField.css

```diff
@@ -44,7 +44,6 @@
 
   display: flex;
   width: 100%;
-  height: var(--field--height);
```

Comment:
this was restricting the height of the container that can now be a lot more than just the input field itself

I'm pretty sure this is safe to remove, but I'm really not too sure what the initial need was for it since presumably you'd be able to let the content determine how tall it needs to be from the get go

Reply:
looking back at this comment now is funny

how little I knew about how important that height was